src/foundry/client-esm/applications/_types.d.mts

```diff
@@ -13,7 +13,7 @@ export {};
 // is also a class in `src/foundry/common/config.d.mts`. It's also just nice to
 // have as reference to keep us synced with the latest version of Foundry.
 
-interface ApplicationConfiguration extends ApplicationV2.Configuration {}
+export interface ApplicationConfiguration extends ApplicationV2.Configuration {}
```

Comment:
it's intentional that this is *not* exported and instead people should use `ApplicationV2.Configuration`

Reply:
Thanks.